engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentRejectionTest.java

```diff
@@ -265,22 +265,17 @@ public void shouldRejectDeploymentIfNoExecutableProcess() {
   @Test
   public void shouldDoAtomicDeployments() {
     // given
-    final String process1Id = "process1";
-    final BpmnModelInstance process1 =
-        Bpmn.createExecutableProcess(process1Id)
-            .startEvent()
-            .businessRuleTask("invalid_business_rule_task_id")
-            .endEvent()
-            .done();
-    final String process2Id = "process2";
-    final BpmnModelInstance process2 =
-        Bpmn.createExecutableProcess(process2Id).startEvent().task().endEvent().done();
+    final String invalidProcessId = "invalid_process_without_start_event";
+    final BpmnModelInstance invalidProcess = Bpmn.createExecutableProcess(invalidProcessId).done();
+    final String validProcessId = "valid_process";
+    final BpmnModelInstance validProcess =
+        Bpmn.createExecutableProcess(validProcessId).startEvent().task().endEvent().done();
```

Comment:
🔧 You could consider inlining the process id strings, as they are not reused and distract a bit from what is being tested. They also don't provide much value in being named specifically because the `Bpmn.createExecuteableProcess(String)` method is used in almost every test and most readers know that this String is a process id.